src/App.js

```diff
@@ -13,14 +18,39 @@ function App() {
   // the "5" button, or the operator if they click one of those buttons) and then call your setter function to update state.
   // Don't forget to pass the functions (and any additional data needed) to the components as props
 
-  return (
-    <div className="container">
-      <Logo />
-      <div className="App">
-        {/* STEP 4 - Render your components here and be sure to properly import/export all files */}
-      </div>
-    </div>
-  );
+ 
+  const [currentNum, setCurrentNum] = useState(" ");
```

Comment:
Instead of an empty string, it apppears that you're using a space?  An empty string or even a 0 might be more appropriate here